docs/auth/README.md

```diff
@@ -0,0 +1,116 @@
+# Running Lighthouse on Authenticated Pages
+
+Standard usages of Lighthouse processes a page as a new user, with no previous session or storage data. This means that pages requiring authenticated access do not work without some additional setup. There are multiple ways to run Lighthouse on an authenticated page - this document focuses on the most flexible approach (Puppeteer), but mentions other approaches briefly at the end.
+
+If you just want to view the code, see `./example-lh-auth.js`.
+
+## The Example Site
+
+There are two pages on the site:
+
+1. `/` - the homepage
+2. `/dashboard`
+
+The homepage shows the login form, but only to users that are not signed in.
+
+The dashboard shows a secret to users that are logged in, but shows an error to users that are not.
+
+The server responds with different HTML for each of these pages and session states, so there are four different pages that must have passable Lighthouse SEO scores.
+
+You can run this server locally if you like:
+
+```sh
+# be in root lighthouse directory
+yarn # install global project deps
+cd docs/auth
+yarn # install deps related to just this documentation
+yarn start # start the server on http://localhost:8000
+```
+
+## Puppeteer
+
+We will use Puppeteer to open a Chrome browser and setup our state by filling and submitting a login form. We'll designate a specific port for Chrome to open a debugging session, so that we can pass the same port to Lighthouse.
+
+To launch Chrome:
+```js
+const PORT = 8041;
+const browser = await puppeteer.launch({
+  args: [`--remote-debugging-port=${PORT}`],
+});
+```
+
+Open the homepage, where the login form is:
+```js
+const page = await browser.newPage();
+await page.goto('http://localhost:8000');
+```
+
+Given a login form that looks like this:
```

Comment:
```suggestion
The homepage contains a form with this DOM tree:
```

Reply:
i don't like this. Do you not like the word `given`?

also, and this is gonna be pedantic so plz dont hate me: this isn't a DOM tree - DOM is the model in memory, which is represented textually via html. or it could be xml. etc.

Reply:
Yeah I really don't like the word `given` here.  I think it should be more simply worded.

Lol, fine, `The homepage contains a web form with this layout:`

Reply:
fwiw, I prefer connor's original text to this new one. "a web form with this layout" sounds like it's trying to be technical but getting it wrong because that's not a layout below it

`Given a login form like this:` seems fine to me 🤷‍♂